### ISSUE TITLE
linux: when player starts playing, stuck in playerState buffering

### ISSUE THREAD

**Issue by esiqveland** [bdlukaa/just_audio_libwinmedia#9]
Hello, thank you for your fantastic work on this. It even works great on linux!

When I play a url using `player.setAudioSource()` the `playbackEventStream` gets stuck showing:

`playing=true state=buffering`

When I call .pause() --> .play() it correctly goes to state:

`playing=true state=ready`

I think it might be because the buffering subject is published after playing == true:

https://github.com/bdlukaa/just_audio_libwinmedia/blob/main/lib/src/just_audio_libwinmedia_real.dart#L65

Or because the subject doesn't update to state `ready` when buffering == false.

**Comment by trapeye** [comment]
did you found solution?

**Comment by esiqveland** [comment]
No, I created a workaround where I on linux assume (override) state == ready when playing starts.

**Comment by trapeye** [comment]
Sorry, how do you override this package, the method I know is fork this repository and edit it to my repository

can you help me give example about it? or link to override package code?

**Comment by esiqveland** [comment]
Sorry, I mean I override this specific condition in the internal state of my own app. I didn't touch this library directly, but changed how I model the internal state on linux in my own app.

**Comment by trapeye** [comment]
Did you get this problem also? how to know when sound is completed, right now the state is incorrect. The windows work properly but not on Linux
